src/hydra/addParameters.js

```diff
@@ -0,0 +1,50 @@
+import Parameter from "../Parameter";
+import fetchResource from "./fetchResource";
+
+export default api => {
+  const promises = [];
+
+  for (const resource of api.resources) {
+    const promise = fetchResource(resource.url).then(response => {
+      if (!response.parameters) {
+        return [];
+      }
+
+      const resourceParameters = [];
+      const { parameters = {} } = response;
+
+      parameters.forEach(parameter => {
```

Comment:
You only used `parameter.variable`, `parameter.property` and `parameter.required`. Maybe could you use destructuring?

```js
parameters.forEach(({ property = null, required, variable }) => {
    if (null === property) {
        return;
    }

    // ...

    resourceParameter.push(
        new Parameter(variable, range, required, "")
    );
});
```